signcrypt_seal.go

```diff
@@ -226,9 +226,13 @@ func checkSigncryptReceivers(receiverBoxKeys []BoxPublicKey, receiverSymmetricKe
 	return nil
 }
 
-func shuffleSigncryptReceivers(receiverBoxKeys []BoxPublicKey, receiverSymmetricKeys []ReceiverSymmetricKey) []receiverKeysMaker {
+func shuffleSigncryptReceivers(receiverBoxKeys []BoxPublicKey, receiverSymmetricKeys []ReceiverSymmetricKey) ([]receiverKeysMaker, error) {
```

Comment:
It looks like this interface change is propagating pretty far, just to deal with possible failures in the RNG. That should more or less never happen, right? It might be cleaner to just make them panics.

Reply:
I agree that it should more or less never happen. Not sure about making this one particular case panic, though -- everywhere else, we handle and propagate errors from `crypto.rand/Reader`.